src/test/scala/com/fulcrumgenomics/fasta/SortSequenceDictionaryTest.scala

```diff
@@ -0,0 +1,230 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2022 Fulcrum Genomics LLC
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.fulcrumgenomics.fasta
+
+import com.fulcrumgenomics.commons.io.PathUtil
+import com.fulcrumgenomics.testing.UnitSpec
+
+class SortSequenceDictionaryTest extends UnitSpec {
+  private val dir        = PathUtil.pathTo("src/test/resources/com/fulcrumgenomics/fasta")
+  private val chr1 = SequenceMetadata(
+    name             = "chr1",
+    length           = 100,
+    aliases          = Seq( "chr1_1", "chr1_2", "chr1_3"),
+  )
+
+  private val chr2 = SequenceMetadata(
+    name             = "chr2",
+    length           = 100,
+    aliases          = Seq( "chr2_1", "chr2_2", "chr2_3"),
+  )
+  private val chrM = SequenceMetadata(
+    name             = "chrM",
+    length           = 100,
+    aliases          = Seq( "chrM_1", "chrM_2", "chrM_3"),
+  )
+
+  private val chr1Alias = SequenceMetadata(
+    name="chr1_1",
+    length=100,
+  )
+  private val chr2Alias = SequenceMetadata(
+    name="chr2_1",
+    length=100,
+  )
+  private val chrMAlias = SequenceMetadata(
+    name="chrM_1",
+    length=100,
+  )
+
+  private val baseDictionary = SequenceDictionary(chr1, chr2, chrM)
```

Comment:
I think this would stop me from having to remember what's in baseDictionary in tests below
```suggestion
  private val dictChr1Chr2ChrM = SequenceDictionary(chr1, chr2, chrM)
```